pkg/sql/sem/tree/show.go

```diff
@@ -1119,8 +1121,56 @@ func (node *ShowFingerprints) Format(ctx *FmtCtx) {
 		ctx.WriteString("SHOW EXPERIMENTAL_FINGERPRINTS FROM VIRTUAL CLUSTER ")
 		ctx.FormatNode(node.TenantSpec)
 	}
+
+	if !node.Options.IsDefault() {
+		ctx.WriteString(" WITH OPTIONS (")
+		ctx.FormatNode(&node.Options)
+		ctx.WriteString(")")
+	}
 }
 
+// ShowFingerprintOptions describes options for the SHOW EXPERIMENTAL_FINGERPINT
+// execution.
+type ShowFingerprintOptions struct {
+	StartTimestamp Expr
```

Comment:
It would be good to get confirmation on whether we need to update any Walk functions now that we've added an Expr.  Perhaps sql foundations can be of some assistance.

Reply:
the only think you should need it to make sure the new ShowFingerprintOptions expr implements the `Walk` function.

Reply:
i dont think `ShowFingerprints` implemented the walkable interface. I added this in.